ui/partials/main/thActiveFiltersBar.html

```diff
@@ -1,20 +1,27 @@
 <!-- Show this certain filters are active in Treeherder -->
-<div class="alert-info active-filters-bar">
+<div ng-if="newFieldFilter || filterBarFilters.length"
+     class="alert-info active-filters-bar">
     <div ng-show="filterBarFilters.length" ng-cloak>
-      <i class="fa fa-info-circle" aria-hidden="true"></i>
-      <span><b>Active Filters</b></span>
-
-      <span ng-repeat="filter in filterBarFilters" class="btn btn-sm"
-            title="Click to clear the {{ filter.field }} filter"
-            ng-click="jobFilters.removeFilter(filter.key, filter.value)">
-          <span ng-if="filter.field == 'author'">{{ filter.field }}: {{filter.value.split('@')[0] | limitTo: 20}}</span>
-          <span ng-if="filter !== 'author'">{{ filter.field }}: {{filter.value | limitTo: 12}}</span>
-          <i class="fa fa-times-circle"></i>
+      <span class="active-filters-title">
+        <i class="fa fa-info-circle" aria-hidden="true"></i>
+        <b>Active Filters</b>
+      </span>
+      <span ng-repeat="filter in filterBarFilters"
+            class="filtersbar-filter">
+          <span title="Filter by {{ filter.field}}: {{ filter.value }}"
+                ng-if="filter.field == 'author'"><b>{{ filter.field }}:</b> {{filter.value.split('@')[0] | limitTo: 20}}</span>
+          <span title="Filter by {{ filter.field}}: {{ filter.value }}"
+                ng-if="filter !== 'author'"><b>{{ filter.field }}:</b> {{filter.value | limitTo: 12}}</span>
+          <span class="pointable"
+                title="Click to clear {{ filter.field }}"
+                ng-click="jobFilters.removeFilter(filter.key, filter.value)">
```

Comment:
And I did the same here as we do with classification annotations (and other filters I've seen out on the web) where only the [x] icon closes the filter, rather than the whole filter string acting as a target area to close. I defer to you guys though, maybe @KWierso has an opinion on this. It may be annoying to have to find the close icon each time, vs. the entire string.

Reply:
No, you're right.  this was just a mistake on my part.  Thanks for fixing it.  :)